mmdeploy/backend/rknn/wrapper.py

```diff
@@ -67,3 +67,22 @@ def forward(self, inputs: Dict[str,
     def __rknnnn_execute(self, inputs: Sequence[np.array]):
         """Run inference with RKNN."""
         return self.rknn.inference(inputs)
+
+    def output_to_list(
```

Comment:
Why don't we enforce forward return a dict.

Reply:
Returning a dictionary is meaningless for the RKNN backend since it returns a list instead of a dictionary.

Reply:
It is the expected behavior of the wrapper.

Reply:
I can return a dictionary then. But the key would be meaningless for users and may cause confusion.

Reply:
Did rknn change the output order?

Reply:
Nope. I would set keys to string of numbers.

Reply:
Use the output names in the onnx as the key.

Reply:
RKNN model has its own output names that are not ONNX out names.

Reply:
Or, we can align the behavior to SDK such as [torchscript](https://github.com/open-mmlab/mmdeploy/blob/522fcc06350d426df0a152160b795507ad7982fc/csrc/mmdeploy/net/torchscript/torch_net.cpp#L126)